internal/mq/nats.go

```diff
@@ -85,16 +90,26 @@ func New(ctx context.Context, uri string, root string, maxPending int, db *dao.D
 		ctx:        ctx,
 		uri:        uri,
 		root:       root,
-		ns:         fmt.Sprintf("%s.%s", root, db.DatabaseName()),
 		db:         db,
 		streams:    map[string]jetstream.Stream{},
 		publishBus: make(chan msg, 1),
 		doneChan:   make(chan struct{}),
 		snapsDone:  make(chan struct{}),
 	}
 
+	hostname, err := os.Hostname()
+	if err != nil {
+		return nil, err
+	}
+	hostname = strings.Split(hostname, ".")[0]
+
+	opts := []nats.Option{
+		nats.Name(hostname),
+		nats.PingInterval(2 * time.Second),
```

Comment:
Until we handle recreation of the nats client, I think perhaps also add this option:

`nats.MaxReconnects(-1)` 

which sets unlimited reconnects